### PR TITLE
feat: add integration test

### LINKED ISSUE #316: feat: support integration test for java/python sdk and offline batch
**Is your feature request related to a problem? Please describe.**

Add integration test for python/java sdk and offline batch 
**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by wuyou10206**
* **Please check if the PR fulfills these requirements**
support integration test for java/python sdk and offline batch

### CODE REVIEW

**Comment by jingchen2222** [comment]
Config workflow for integration test. 
Recommend to run integration test manually.

**Comment by jingchen2222** [comment]
openmldb-jmh has been added by https://github.com/4paradigm/OpenMLDB/pull/367.

File: test/integration-test/openmldb-test-java/openmldb-test-common/src/main/resources/fedb_deploy.properties [hunk 1]
```diff
@@ -0,0 +1,10 @@
+
+#zk的url
+zk_url=http://pkg.4paradigm.com:81/rtidb/test/zookeeper-3.4.14.tar.gz
```
Comment (aceforeverd) [inline test/integration-test/openmldb-test-java/openmldb-test-common/src/main/resources/fedb_deploy.properties]: inner link should removed

File: test/integration-test/openmldb-test-java/openmldb-sdk-test/src/main/resources/command.properties [hunk 2]
```diff
@@ -0,0 +1,5 @@
+
+remote_ip=172.24.4.55
+remote_user=zhaowei01
+remote_password=1qaz0p;/
```
Comment (aceforeverd) [inline test/integration-test/openmldb-test-java/openmldb-sdk-test/src/main/resources/command.properties:5]: maybe add a comment for how to change the password

**Review by dl239 — APPROVED** [review]

**Review by vagetablechicken — APPROVED** [review]

**Pull request merged**